### PR TITLE
Initial instructions

### PR DESCRIPTION
**Pull request by tatyree**
This is my proposed basic, full stack developer test.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,48 @@
-# technicalTest
-The baseline technical test for candidates interested in working for the UK Judicial Appointments Commission
+# JAC Technical Test
+The baseline technical test for developers interested in working for the UK Judicial Appointments Commission
```
Comment (ollietreend) [inline README.md]: No need for the 'UK' to prepend 'Judicial Appointments Commission'. It's never referred to as 'UK Judicial Appointments Commission'.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,48 @@
...
+
+## Background
+
+The JAC needs a new online tool to allow users to register their interest in a job or jobs.
+
+You have just picked up the ticket to do this piece of work. It needs to be done to a production-ready standard.
+
+## Requirements
+
+We need a webpaage that allows a user to enter their email address and pick from a list of upcoming jobs they wish to apply for.
```
Comment (ollietreend) [inline README.md]: Typo – `webpaage` has two `a`s

File: README.md [hunk 3]
```diff
@@ -1,2 +1,48 @@
...
+
+The jobs are as follows:
+
+| Job opens in | Job Title |
+| --- | --- |
+| 30 days | District Judge |
+| 60 days | Tribunal Member |
+| 65 days | Adjudicator |
+
++ We need a to maintain a record of which users applied for which jobs for 90 days after the job opening has closed.
```
Comment (ollietreend) [inline README.md]: Given you've said a list needs to be maintained until 90 days after closing, should the jobs table above also contain a closing date? Or, what are you expecting (if anything) to see in relation to this point?

Comment (tatyree) [reply README.md]: Yes

**Pull request merged**